### ISSUE TITLE
Implement Stalinsort instead of Timsort by default

### ISSUE THREAD

**Issue by egorgam** [jokteur/python_communism#16]
![image](https://user-images.githubusercontent.com/13712792/149671362-1811d240-4f31-4dfa-afb6-926ed5eef626.png)

**Comment by kmmbvnr** [comment]
Killing random elements until list became sorted?

**Comment by jokteur** [comment]
No, not random, but if you encounter a value that is not in order, then the value should be eliminated.

**Comment by arduinka55055** [comment]
NOPE. implement Leninsort, Stalinsort, Khrushchevsort and brezhnevsort
use all of them to make real communism. but beware of putinsort

**Comment by egorgam** [comment]
![Сталин 47-й год](https://user-images.githubusercontent.com/13712792/150656406-7414de91-3b84-4288-a545-ef0d1421ceb3.gif)

arduinka55055.sorted()
